handler/src/test/java/io/netty/handler/ssl/SniHandlerTest.java

```diff
@@ -368,10 +368,12 @@ public void testMajorVersionNot3() throws Exception {
 
             ch.close();
 
-            // When the channel is closed the SslHandler will write an empty buffer to the channel.
-            ByteBuf buf = ch.readOutbound();
-            if (buf != null) {
-                assertFalse(buf.isReadable());
+            // Consume all the outbound data that may be produced by the SSLEngine.
+            for (;;) {
```

Comment:
Most Java developers seem to prefer 'while loop' to 'for loop' when writing infinite loops.

I referred to [this](https://stackoverflow.com/questions/15989618/java-infinite-loop-convention).